src/nvim/normal.h

```diff
@@ -72,6 +72,8 @@ typedef struct cmdarg_S {
 #define CA_COMMAND_BUSY     1   // skip restarting edit() once
 #define CA_NO_ADJ_OP_END    2   // don't adjust operator end
 
+extern bool first_redraw;
```

Comment:
follow existing conventions

```suggestion
extern bool did_first_redraw;
```